src/components/Pages/Home.js

```diff
@@ -0,0 +1,86 @@
+// @ts-nocheck
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { loadContinentData } from '../../redux/countries/countries';
+import CountryCard from '../Country/CountryCard';
+import africaImage from '../../assets/africa.png';
+import classes from './Home.module.css';
+
+const Home = () => {
+  const countries = useSelector((state) => state.countries);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(loadContinentData());
+  }, []);
+
+  const totalVacinated = () => {
+    let total = 0;
+    if (countries) {
+      countries.forEach((country) => {
+        total += country.All.administered;
+      });
```

Comment:
- [ ] **Optional:** Kindly change the `#forEach` to `map`. This is because `forEach` method gives an abnormal behavior in `ReactJS`. Implement it throughout your project if you do consider it.